.github/workflows/lint.yml

```
@@ -0,0 +1,50 @@
+name: Code Linting
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - '.github/workflows/lint.yml'
+      - '**.js'
+      - '**.json'
+      - '**.mjs'
+      - '.eslint*'
+      - '.nvmrc'
+      - '**/package.json'
+      - 'package-lock.json'
+  pull_request:
+    branches:
+      - main
+      - 'feature/**'
+    paths:
+      - '.github/workflows/lint.yml'
+      - '**.js'
+      - '**.json'
+      - '**.mjs'
+      - '.eslint*'
+      - '.nvmrc'
+      - '**/package.json'
```

Not a blocker just for information. The `package.json` is in root folder so why we use start before it? 

```suggestion
      - 'package.json'
```

---

I took this from the Performance Lab plugin workflow which has this. Not exactly sure why, but let's keep it as is for now.